src/svg-quirks-mode/svg-renderer.js

```diff
@@ -73,6 +73,13 @@ class SvgRenderer {
     }
 
     /**
+     * @return {[number,number]} the natural size, in Scratch units, of this SVG.
+     */
+    get size () {
```

Comment:
The only thing that seems particularly obvious to me is `scratchSize` – since it's in "Scratch units", would that be better?